### DISCUSSION TITLE
Can`t connected Customer portal2 to Yetifirce 6.3.0

### DISCUSSION THREAD

**Discussion by Rukatekirin** [YetiForceCompany/YetiForceCRM#15630] [How to] [answered]
Hello!
We can`t connect CustomerPortal2 via webservice standart to Yetiforce 6.3.0.
All installation made according to instruction https://yetiforce.com/en/knowledge-base/documentation/implementer-documentation/item/how-to-install-yetiforce-portal
On the final step we recived next message:
"PUT /webservice/WebservicePremium/Install HTTP/1.1" 404 472 "-" "YetiForce Portal | Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko)
We add api key for webservice standart
What we need to do for sucsessfull connection?
Thank You.

**Comment by bpabiszczak** [comment, marked as answer]
Portal2 work only with API Premium [webservice premium].

**Reply by Rukatekirin** [reply]
@bpabiszczak Thank You.
